packages/harpguru-core/src/utils/reduce-for-new-column-bounds/determine-best-7-holes.ts

```diff
@@ -0,0 +1,14 @@
+export const determineBest7Holes = (
+  harpLength: number,
+  inputColumnBounds: readonly [number, number]
+): readonly [number, number] => {
+  const [startColumn, endColumn] = inputColumnBounds
+  const boundsLength = endColumn - startColumn + 1
+  if (harpLength <= boundsLength) return [0, 6] as const
+  if (endColumn < harpLength) return inputColumnBounds
+  if (endColumn >= harpLength) {
+    const difference = endColumn - harpLength + 1
+    return [startColumn - difference, endColumn - difference]
+  }
+  throw Error('temporary error')
```

Comment:
Missing TODO here. Don't forget to fix the error message.

Reply:
Is an error message even necessary now?

Reply:
This function has gone and has been replaced with something more generic and not requiring an Error to be thrown (determine-best-bounds-for-harp-length)